### DISCUSSION TITLE
The problem of build module.scss to css

### DISCUSSION THREAD

**Discussion by jamiecodejob** [vercel/next.js#23390] [Help]
Hi everyone,

I encountered a problem when I use npm run build within NEXT.js. I can not get index.css although I written the rule such as

path = require('path')

module.exports = {
  sassOptions: {
    includePaths: [path.join(__dirname, 'styles')],
  },
}

Because of the above problem, I cannot generate css file, my website lost its style.

If you have any idea, please tell me how to fix this problem.
Thank you for your helping

**Comment by Sehyn** [comment]
Same problem.
